### PR TITLE
OSDOCS-4319: Removed Recommended install practices from Scalability and performance guide

### PR DESCRIPTION
**Pull request by ahardin-rh**
Version(s):
All supported versions, 4.8+

Issue:
https://issues.redhat.com/browse/OSDOCS-4319

Link to docs preview:
https://53335--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/recommended-host-practices.html (note removed content)

QE review:
Content removal. No QE required.

No redirect required per discussion with Kathryn.

### CODE REVIEW

File: welcome/learn_more_about_openshift.adoc [hunk 1]
```diff
@@ -30,9 +30,6 @@ Use the following sections to find content to help you learn about and use {prod
 |
 | xref:../backup_and_restore/index.adoc#backup-restore-overview[Backup and restore]
 |
-
-|
-| xref:../scalability_and_performance/recommended-install-practices.adoc#recommended-cluster-install-practices[Performance and scalability]
```
Comment (jldohmann) [inline welcome/learn_more_about_openshift.adoc:35]: I'm having trouble finding this table link in any of the currently built versions of the doc, so I can't compare the tables before and after. Could you link me? 🙂

**Review by jldohmann — COMMENTED** [review]
Mostly LGTM I left a question below

File: welcome/learn_more_about_openshift.adoc (hunk 1 above)
Comment (ahardin-rh) [reply welcome/learn_more_about_openshift.adoc:35]: @jldohmann So strange! I can't seem to find it either. :thinking: It's not in any of the published docs.

**Pull request merged**